ONSdigital/design-system
# Stop save cookies button default behaviour in example

### What is the context of this PR?
Fixes: #1089
- Adds `dsExample` parameter to button to allow overriding of default button behaviour in forms for DS examples
- Sets `dsExample` parameter to the global variable `isPatternLib`
- Updates name of current `DSExample` param for error panels to align with other params naming convention
- Removes some whitespace and fixes some indenting
- Fix console error when clicking error header example links

### How to review 
- Click the "Save changes" button on the control cookies example and see that it doesn't take you to the previous page
- Click the error header example links and see there is no "scrollIntoView" error in console and that links still work in error prototype
